src/components/AddBook.js

```diff
@@ -0,0 +1,49 @@
+import { React, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { nanoid } from '@reduxjs/toolkit';
+import { addBook } from '../redux/books/books';
+
+function AddBook() {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  function addTitle(e) {
+    setTitle(e.target.value);
+  }
+
+  function addAuthor(e) {
+    setAuthor(e.target.value);
+  }
```

Comment:
- [ **OPTIONAL** ] For best practices and clean code, kindly make sure you update the above functions by using `Arrow functions` instead of `function` keyword

Reply:
Thank you @tresorsawasawa, I have changed the functions to the arrow function.